src/lib/primitives.unit-test.ts

```diff
@@ -19,5 +19,6 @@ let keypair = Primitives.generateKeypair();
 let proof = Primitives.prove([], [], keypair);
 let ok = Primitives.verify([], keypair.verificationKey(), proof);
 expect(ok).toEqual(true);
+shutdown();
```

Comment:
can we move this to the very bottom, so that the success line is displayed? :D

Reply:
My bad, wasn't thinking there :sweat_smile: